File: dart/variables/5.dart
import 'dart:io';

void main() {
  print("Calcular el 20% de cualquier número");

  print("Digite un número: ");

  int num1 = int.parse(stdin.readLineSync()!);

  double total = num1 * 0.2;

  print("El 20% de ${num1} es igual a ${total}");
  
}

File: variablesCsharp/5.cs
using System;

namespace Variables
{
    internal class Cinco
    {
        public Cinco() {
            //20% de cualquier digito
            Console.WriteLine("Digite un número: ");
            int num = int.Parse(Console.ReadLine());

            double resultado = num * 0.2;

            Console.WriteLine("El 20% de " + num + " es igual: " + resultado);
        }
    }
}
